File: loianeTraining/javaBasico/exercicios-aulas-18-19/Ex11.java
public class Ex11 {
    public static void main(String[] args) {
        int[] a = { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 };

        var quantityEven = 0;
        for (int i = 0; i < a.length; i++) {
            if (a[i] % 2 == 0) {
                quantityEven++;
            }
        }
        System.out.println("Quantidade de pares: " + quantityEven);
    }
}

File: Loiane_Training_C#/Basico/E/Ex11.cs
using System;

class Ex11
{
    static void Main(string[] args)
    {
        double[] vetorA = new double[10] { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 };

        int contadorDePares = 0;
        for (int i = 0; i < vetorA.GetLength(0); i++)
        {
            if (vetorA[i] % 2 == 0)
            {
                contadorDePares++;
            }
        }

        Console.Write("\nVetor A: ");
        for (int i = 0; i < vetorA.GetLength(0); i++)
        {
            Console.Write(vetorA[i] + " ");
        }

        Console.Write("\t\tVetor A tem {0} pares", contadorDePares);
    }
}